src/main/java/seedu/duke/Parser.java

```diff
@@ -69,7 +72,7 @@ public static ArrayList<String> parseCommand(String userInput) {
      * @param uniModuleList The corresponding ArrayList of that specified uni.
      * @param database      Database of the user's saved list of modules.
      */
-    public static boolean deleteModule(int indexToDelete, ArrayList<Module> uniModuleList,
+    public static boolean handleDeleteModule(int indexToDelete, ArrayList<Module> uniModuleList,
```

Comment:
I think it would be better to move this to Storage Class since Parser just does preparation, execution should be at Storage. The try catch portion I think can put in Storage, then the index one can put in Parser